### DISCUSSION TITLE
When is Excel support coming back for exporting notifications?

### DISCUSSION THREAD

**Discussion by darylknight** [solspace/craft-freeform#464] [Feature Requests]
A client is just starting to look into exporting their form submissions. Once of the reasons we got them to purchase the Pro version of freeform was for exporting to Excel, but it's been removed [since April](https://docs.solspace.com/craft/freeform/v4/setup/changelog.html#_4-0-0-beta-4-2022-04-28). Any news when it's coming back?

**Comment by kjmartens** [comment]
Thanks for your feedback, @darylknight.

We do have plans to add it back, but the issue is that the library we used to handle this is using a version of `psr/simple-cache` that is no longer compatible with Craft 4. All users will run into an error when trying to install Freeform. The author of the [PHPOffice/PhpSpreadsheet](https://github.com/PHPOffice/PhpSpreadsheet) library is hesitant to support `psr/simple-cache` 3.0, but may add support for it in v2 of their `PHPOffice/PhpSpreadsheet` library. That is yet to be announced though.

More on the issue here:
https://github.com/PHPOffice/PhpSpreadsheet/issues/2790

We are also looking to see if there's another library that might be able to handle this, but they all seem abandoned. 😞

**Comment by darylknight** [comment]
Thanks for the update Kelsey. That is a shame! Perhaps in the meantime, it might be a good idea to remove references to this in the documentation as it's no longer possible?

**Comment by kjmartens** [comment]
I guess I never updated this, but we did add support back for this in Freeform 4.0.9 (2022-12-09) 🙂

**Comment by darylknight** [comment]
Awesome, thank you :)
